### DISCUSSION TITLE
Moving more than one picture?

### DISCUSSION THREAD

**Discussion by TMaekler** [AlUlkesh/stable-diffusion-webui-images-browser#247] [Ideas]
Hi,

this is a helpful addon. Thanks for that.

With 'delete next x pictures' it's easy to delete a bunch of them. Could you expand that with 'move next x pictures to folder' so we can move more than one at a time? We could do that manually, but when you move them to an 'others' folder, you lose the ranking that you gave those pictures.

**Comment by AlUlkesh** [comment]
Hi,

Thanks. I'm actually thinking of moving towards real selections for multi-image actions, once this is possible: #245

If you move images manually to an "Others" folder, you should be able to use `Image-Browser - Maintenance` -  `Reapply ranking after moving files`.
